### DISCUSSION TITLE
Collection of open source Zepp OS works

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#6] [General]
Zepp OS has been open for more than a year, and developers from all over the world have produced a lot of great work.

A lot of this code is open source, and we are now collecting these works from everyone to be able to showcase themselves and provide a guide for other people in the community.

Just reply to this discussion with the project name, a brief description, and the project address.

Or submit a pr or issue to our awesome project https://github.com/zepp-health/awesome-zeppos.

Thanks again to all the developers for their contributions to the Zepp OS ecosystem.

**Comment by roigreenberg** [comment]
Hi.
Can you make the Android app an open source too?
I would like to be able to add features there too.

**Reply by MyoungXUE** [reply]
Hi, thanks for supporting Zepp OS technical forum!

There is no open source plan for Zepp App, so if you have any suggestions for features, you can mention them to us in Developer Feedback category.

**Comment by fwz233** [comment]
一个txt文件查看项目https://github.com/fwz233/txtview
😂

**Reply by htoooth** [reply]
@fwz233 welcome to pr for https://github.com/zepp-health/awesome-zeppos project.

**Comment by XiaomaiTX** [comment]
A library for providing simple animations in ZeppOS. 
一个用于在ZeppOS中提供简单动画的库

https://github.com/XiaomaiTX/zeppos-fx

**Comment by MoralGEW** [comment]
Hello, can you provide a smart keyboard to API 3.0?
